DEVELOPMENT.md

```diff
@@ -20,34 +26,30 @@ After cloning suomifi-ui-components, run `yarn` to fetch its dependencies. Then,
 
 9. `yarn bundle-analyzer` shows analyzation of bundle size.
 
-
-
 ❗️After you pull changes from the repo, remember to run `yarn` to make sure that you have all the needed dependencies installed on your setup.
 
 ## Source
 
-[![suomifi-ui-components architecture diagram](suomifi-ui-components.png 'suomifi-ui-components architecture')](suomifi-ui-components.png)
-
 Source contains 3 stages of components:
 
 1. `reset` is for resetting html tags (don't do too opinionated stuff here).
-2. `components` are accessible/a11y version of components and HTML-semantics without Suomi.fi related.
+2. `components` are accessible/a11y version of components and HTML-semantics without Suomi.fi related. The components level is being removed, and is not supported any longer.
 3. `core` contains Suomi.fi-styleguide as theme, components and CSS exports.
 
 - _Export `src/core`-components at `src/core/index` and `src/index`._
 - Export `src/components`-components at `src/components/index`.
 
 **Don't do relative imports from `src/index`, use those 3-levels export locations.**
 
-**Do not create duplication of source or styles for component, use syntax that can be used in exports.**
+**Do not create duplication of source or styles for component.**
 
 Export interfaces for exported functions/components. Typescript will generate declaration files from exported interfaces (.d.ts). Write comments/documentatiion to all properties that need to be shown at styleguide.
 
 ### Props that are passed on to another component
 
 ```jsx
-// put rest of the props to passProps, but takeAwayProp and takeAanother
-const { takeAwayProp, takeAanother: renamedAsProp, ...passprops } = withSuomifiDefaultProps(this.props);
+// put rest of the props to passProps, but takeAwayProp and takeAnother
```

Comment:
=> destructure takeAway and takeAnother props and put rest of the props to passProps.

Reply:
That is clearer, I'll replace it. 👍